### PR TITLE
docs(features): update stronghold page

### PR DESCRIPTION
**Pull request by pewsheen**
#### What kind of changes does this PR include?

- New or updated content
  - Update `features/stronghold` page

#### Description

- What does this PR change? Give us a brief description. 
  - Update `features/stronghold` page

### CODE REVIEW

**Comment by vasfvitor** [comment]
just curious, have you tested all those features yourself? I just want to be sure that the guide works as intended

**Comment by pewsheen** [comment]
> just curious, have you tested all those features yourself? I just want to be sure that the guide works as intended

Yeah, I've run through all those features

File: src/content/docs/features/stronghold.mdx [hunk 1]
```diff
@@ -3,12 +3,232 @@ title: Stronghold
...
+  npm="npm run tauri add stronghold"
+  yarn="yarn run tauri add stronghold"
+  pnpm="pnpm tauri add stronghold"
+  cargo="cargo tauri add stronghold"
+/>
+
+  </TabItem>
+  <TabItem label = "Manual">
+
+1. Install the stronghold plugin by adding the following to your `Cargo.toml` file:
```
Comment (dreyfus92) [inline src/content/docs/features/stronghold.mdx]: Would you mind using the `<Steps>` component for this part?

**Review by dreyfus92 — COMMENTED** [review]
Hey @pewsheen, awesome job. I just left a small suggestion 😁

File: src/content/docs/features/stronghold.mdx (hunk 1 above)
Comment (pewsheen) [reply src/content/docs/features/stronghold.mdx]: Sure! It looks better 👍

**Review by dreyfus92 — APPROVED** [review]
Thank you @pewsheen LGTM ✅

**Comment by dklassic** [comment]
Closes #1492

**Pull request merged**